fighthealthinsurance/stripe_utils.py

```
@@ -1,17 +1,54 @@
+                stripe_meters = stripe.billing.Meter.list()
+                for candidate in stripe_meters:
+                    if candidate.display_name == product_name:
+                        meter_id = candidate.id
+                        break
+                if meter_id is None:
+                    raise Exception("No meter")
+                meter = StripeMeter.objects.create(
+                    name=product_name,
+                    stripe_meter_id=meter_id,
+                )
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)